docs/rules/structured-imports.md

```diff
@@ -0,0 +1,33 @@
+# Enforces a certain structure in your import statements
+
+## Rationale
+- Consistency in import statement structure throughout the codebase
+
+## Rule Details
+Enforce a certain structure in your imports. Import structure should be listed in the following order:
```

Comment:
Again, this is vague/flowery.

```
Imports should be in the following order:
* Externals
* Absolute paths
* Parent directories
* Siblings
```

Note that I'm borrowing "externals" from the [`eslint import/order` rule](https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/order.md).  Everything in TypeScript is a module, so a more specific term is useful.